program/rust/src/c_oracle_header.rs

```diff
@@ -34,94 +34,128 @@ pub trait PythAccount: Pod {
     }
 }
 
-impl PythAccount for pc_map_table_t {
+impl PythAccount for MappingAccount {
     const ACCOUNT_TYPE: u32 = PC_ACCTYPE_MAPPING;
-    const INITIAL_SIZE: u32 = PC_MAP_TABLE_T_PROD_OFFSET as u32;
+    const INITIAL_SIZE: u32 = 56; // Tested in test_sizes.rs
```

Comment:
I'm trying to avoid `as` everywhere. It's tricky to use `try_from` here because it's a constant. The solution is hardcoding and testing this in `test_size.rs`.

Reply:
Could you add a comment explaining how this value is calculated?